sdk/core/core/src/az_span.c

```diff
@@ -679,3 +679,31 @@ AZ_NODISCARD az_result _az_span_url_encode(az_span destination, az_span source,
 
   return AZ_OK;
 }
+
+AZ_NODISCARD az_span _az_span_token(az_span source, az_span delimiter, az_span* out_remainder)
+{
+  _az_PRECONDITION_VALID_SPAN(delimiter, 1, false);
+  _az_PRECONDITION_NOT_NULL(out_remainder);
+
+  if (az_span_size(source) == 0)
+  {
+    return AZ_SPAN_NULL;
+  }
+  else
```

Comment:
nit:  remove this else from here to avoid the extra indentation

Reply:
Sure, done.